ga4gh/frontend.py

```diff
@@ -195,13 +205,32 @@ def configure(configFile=None, baseConfig="ProductionConfig", extraConfig={}):
     theBackend.setDefaultPageSize(app.config["DEFAULT_PAGE_SIZE"])
     theBackend.setMaxResponseLength(app.config["MAX_RESPONSE_LENGTH"])
     app.backend = theBackend
+    app.secret_key = os.urandom(24)
+    app.oidcClient = oic.oic.Client()
+    if oidcConfig:
+        configDict = yaml.load(open(oidcConfig))
```

Comment:
Close this file after you're done with it.  Best way to do this is with the `with open(oidcConfig) as oidcConfigFile: ...` construct so you don't explicitly call (or forget to call) `close`